poetry.lock

```diff
@@ -19,7 +19,7 @@ python-versions = "*"
 
 [[package]]
 name = "anyio"
-version = "3.6.1"
```

Comment:
Please avoid running `poetry update` in this PR so that we don't have issues with dependencies.